site-src/contributing/index.md

```diff
@@ -102,13 +102,39 @@ service mesh use cases can be found in a separate
 
 | Date           | Title |    |
 |----------------|-------|----|
-| November, 2019 | [Kubecon 2019 San Diego: Evolving the Kubernetes Ingress APIs to GA and Beyond][2019-kubecon-na-slides] | [slides][2019-kubecon-na-slides], [video][2019-kubecon-na-video]|
+| Mar, 2024      | Kubecon 2024 Paris: Configuring Your Service Mesh with Gateway API | [video][2024-kubecon-video-1]|
+| Mar, 2024      | Kubecon 2024 Paris: Gateway API: Beyond GA | [video][2024-kubecon-video-2]|
+| Mar, 2024      | Kubecon 2024 Paris: Gateway API: Beyond GA | [video][2024-kubecon-video-3]|
```

Comment:
```suggestion
| Mar, 2024      | Kubecon 2024 Paris: Tutorial: Configuring Your Service Mesh with Gateway API  | [video][2024-kubecon-video-3]|
```
This one needs a different name.

Reply:
Thanks @youngnick  for review
I will incorporate your suggestion and commit again.